### PR TITLE
[IOPAE-301] client apim config

### PR DESCRIPTION
**Pull request by StefanoDoc**
Add Apim configuration
#### List of Changes
Add config for apim plus some minor fixes

#### How Has This Been Tested?
unit test

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/src/config.ts [hunk 1]
```diff
@@ -32,12 +32,30 @@ export const JiraConfig = t.interface({
 });
 export type JiraConfig = t.TypeOf<typeof JiraConfig>;
 
-// global app configuration
+// Service Principal Creds configuration
+export const ServicePrincipalConfig = t.interface({
```
Comment (balanza) [inline apps/io-services-cms-webapp/src/config.ts:36]: Naming: _ServicePrincipal_ of what? Also: we can use comments to explain **why** we write something, just repeating the code in plain English has usually little value.

File: apps/io-services-cms-webapp/src/jira_client.ts [hunk 2]
```diff
@@ -18,24 +18,25 @@ export const CreateJiraIssueResponse = t.interface({
...
-          comment: t.interface({
-            comments: t.readonlyArray(t.interface({ body: t.string })),
-          }),
-          status: t.interface({
-            name: t.string,
-          }),
-        }),
-      })
-    ),
+    issues: t.readonlyArray(JiraIssue),
```
Comment (balanza) [inline apps/io-services-cms-webapp/src/jira_client.ts:39]: 👍

**Review by balanza — COMMENTED** [review]
Should we add configs to `infra` as well?

File: apps/io-services-cms-webapp/src/config.ts (hunk 1 above)
Comment (gunzip) [reply apps/io-services-cms-webapp/src/config.ts:36]: FYI https://github.com/pagopa/developer-portal/pull/46#discussion_r1179390139
